instance/tasks.py

```diff
@@ -66,6 +67,16 @@ def spawn_appserver(instance_ref_id, mark_active_on_success=False, num_attempts=
 
 
 @db_task()
+def set_appserver_active(appserver_id):
+    """
+    Mark an AppServer as active.
+    """
+    logger.info('Retrieving AppServer: ID=%s', appserver_id)
```

Comment:
No, this line is analogous to https://github.com/open-craft/opencraft/blob/97a3b3a726c35b927706c2286b3b795e2869a262/instance/tasks.py#L54.  Since retrieving the AppServer might fail, we log which AppServer we are trying to retrieve.  At this point, we don't have any instance or AppServer to log to.

Once the AppServer was successfully retrieved, we call `OpenEdXInstance.set_instance_active()`, and the first line inside this message logs to the instance log.  For this reason, I didn't think it's necessary to log to the instance log from here.